tests/compare-sharding.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/bash
+
+trap 'kill 0' SIGTERM
+
+kubectl -n kube-system port-forward deployment/kube-state-metrics 8080:8080 &
+kubectl -n kube-system port-forward pod/kube-state-metrics-0 8082:8080 &
+kubectl -n kube-system port-forward pod/kube-state-metrics-1 8084:8080 &
+kubectl -n kube-system port-forward pod/kube-state-metrics-2 8086:8080 &
+
+sleep 3
+
+RESULT_UNSHARDED="$(mktemp)"
+RESULT_SHARDED_UNSORTED="$(mktemp)"
+RESULT_SHARDED="$(mktemp)"
+
+curl localhost:8080/metrics | grep -v "^#" | sort | tee "${RESULT_UNSHARDED}"
```

Comment:
can we have a ports array and loop through that?

Reply:
I don't think we can, as they are slightly different each time. Also I like the explicitness, and it's just a script for manual verification.

Reply:
I can agree with that.